sdk/core/abort-controller/package.json

```diff
@@ -74,7 +74,7 @@
   "devDependencies": {
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
-    "@microsoft/api-extractor": "7.18.11",
+    "@microsoft/api-extractor": "~7.31.1",
```

Comment:
can we make it a `^` or is that too risky?

Reply:
caret is fine too.